addons/website_slides/controllers/main.py

```diff
@@ -678,9 +678,13 @@ def slide_archive(self, slide_id):
         slide = request.env['slide.slide'].browse(int(slide_id))
         if slide.channel_id.can_publish:
             slide.sudo().active = False
-            return True
+            return {
+                'category_id': slide.category_id.id
+            }
 
-        return False
```

Comment:
You're not supposed to change a method return type in stable.
This looks like an improvement rather than a fix.